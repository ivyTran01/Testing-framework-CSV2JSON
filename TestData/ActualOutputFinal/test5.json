[
 {
  "name":"Ivy T.",
  "age":22.0,
  "ice cream lover":1.0,
  "field4":" 5.2,"
 },
 {
  "name":"Tanzil",
  "age":25.0,
  "ice cream lover":0.0,
  "field4":5.9
 },
 {
  "name":"Danny Yu",
  "age":21.0,
  "ice cream lover":1.0,
  "field4":5.11
 },
 {
  "name":"Abdullah Amjad",
  "age":23.0,
  "ice cream lover":1.0,
  "field4":5.1
 },
 {
  "name":"Christ",
  "age":null,
  "ice cream lover":null,
  "field4":null
 },
 {
  "name":"Christ",
  "age":23.0,
  "ice cream lover":0.0,
  "field4":5.1
 },
 {
  "name":"Monica",
  "age":50.0,
  "ice cream lover":1.0,
  "field4":5.41
 }
]