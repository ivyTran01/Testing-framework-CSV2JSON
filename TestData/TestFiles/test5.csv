name	age	ice_cream_lover
Ivy	" 22	 ""inside"" end "	true
Tanzil	25	false
Danny	21	true
