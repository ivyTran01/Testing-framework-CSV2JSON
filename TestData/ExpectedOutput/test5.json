[
 {
  "name":"Ivy T.",
  "age":22.0,
  "ice cream lover":true,
  "field4":" 5.2,"
 },
 {
  "name":"Tanzil",
  "age":25.0,
  "ice cream lover":false,
  "field4":"5.9"
 },
 {
  "name":"Danny Yu",
  "age":21.0,
  "ice cream lover":true,
  "field4":"5.11"
 },
 {
  "name":"Abdullah Amjad",
  "age":23.0,
  "ice cream lover":true,
  "field4":"5.10"
 },
 {
  "name":"Christ",
  "age":null,
  "ice cream lover":null,
  "field4":null
 },
 {
  "name":"Christ",
  "age":23.0,
  "ice cream lover":false,
  "field4":"5.10"
 },
 {
  "name":"Monica",
  "age":50.0,
  "ice cream lover":true,
  "field4":"5.41"
 }
]